docs/source/video_tutorials.rst

```diff
@@ -0,0 +1,39 @@
+Video Tutorials
+===============
+
+Basics of *stk*
+---------------
+
+https://www.youtube.com/watch?v=mPr9D7nCQ84&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=1
+
+Functional Groups in *stk*
+--------------------------
+
+https://www.youtube.com/watch?v=IaLIN7jXyKQ&list=PLIWYdPQ9hLzVngMF8NOkiApMtgc_ZwZgO&index=2
+
+Cage Building
+-------------
```

Comment:
Overview of organic and metal-organic cage construction in _stk_. This example covers how _stk_ handles isomerism and the concept of hierarchical construction.

Jupyter Notebook is available at https://github.com/andrewtarzia/stk-examples/tree/main/notebooks_from_videos .